component/component.py

```diff
@@ -22,67 +22,76 @@
 an example and will not be supported by TFX team.
 """
 
-from typing import Optional, Text
+from typing import List, Optional, Text
 
 from tfx import types
 from tfx.dsl.components.base import base_component
 from tfx.dsl.components.base import executor_spec
-from tfx.examples.custom_components.hello_world.hello_component import executor
+# from tfx.examples.custom_components.hello_world.hello_component import executor
```

Comment:
We can delete this line completely?
As it refers to the tfx hello world example which we don't need

Reply:
yep yep, will do